src/posix/platform/platform-posix.h

```diff
@@ -305,13 +305,13 @@ void platformNetifProcess(const fd_set *aReadFdSet, const fd_set *aWriteFdSet, c
  * This function initialize virtual time simulation.
  *
  */
-void platformSimInit(void);
+void virtualTimeInit(void);
```

Comment:
how about moving `virtualTimeXXX()` after `platformXXX()` in the header file?

Reply:
There will be duplicated function names.